### DISCUSSION TITLE
Official support for Polylang

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Authors#1216] [Feature Requests]
We should support the Polylang plugin

**Comment by rizaardiyanto1412** [comment]
> Can you try to create two pages as the author, one in each language? At some point the duplicate author profiles should show up.

2 Author Profiles will be created for 1 user

https://wordpress.org/support/topic/polylang-guest-authors-automatically-created-for-each-language/

**Comment by stevejburge** [comment]
Also here https://github.com/publishpress/PublishPress-Authors/issues/1460

**Reply by EmilieCrx** [reply]
We use PublishPress authors pro version 4.4.0 on a multi-site installation with WP version 6.4.3 and are encountering a compatibility problem with the Polylang free plugin version 3.6. When Polylang is activated, the list of articles disappears from the author pages (example.hypotheses.org/author/name).

We had already encountered this problem a few months ago with the free version and mentioned it here : https://github.com/publishpress/PublishPress-Authors/issues/1460

**Comment by stevejburge** [comment]
Closing in favor of https://github.com/publishpress/PublishPress-Authors/issues/1721
